app/build.gradle

```diff
@@ -73,6 +73,7 @@ android {
         }
         nightly {
             initWith release
+            signingConfig signingConfigs.debug
```

Comment:
@eliserichards this slipped my previous review. Let's remove this. :)

In the future, to avoid committing this change, you can use [the same method](https://github.com/mozilla-mobile/fenix#automatically-sign-release-builds) to build with a debug key as Focus.